feat: rework installation page (#22)
argos-ci/docs.argos-ci.com: docs/01-Get Started/visual-testing.mdx

We recommend to lean on an end-to-end testing frameworks to take the screenshots. They facilitate the navigation through website and generally allows to take screenshots in various browsers and screen resolutions. We provide integrations for some of them :

<CiCards>
  <CICardLink href="/cypress">
    <CiCard>
      <CiBody>
        <CiCardImage>
          <img src={CypressLogo} />
        </CiCardImage>
        <CiCardTitle>Cypress</CiCardTitle>
      </CiBody>
      <CiArrow />
    </CiCard>
  </CICardLink>
  <CICardLink href="/puppeteer">
    <CiCard>
      <CiBody>
        <CiCardImage>
          <img src={PuppeteerLogo} style={{ maxHeight: 25, margin: "auto" }} />
        </CiCardImage>
        <CiCardTitle>Puppeteer</CiCardTitle>
      </CiBody>
      <CiArrow />
    </CiCard>
  </CICardLink>
  <CICardLink href="/playwright">
    <CiCard>
      <CiBody>
        <CiCardImage>
          <img src={PlaywrightLogo} />
        </CiCardImage>
        <CiCardTitle>Playwright</CiCardTitle>
      </CiBody>
      <CiArrow />
    </CiCard>
  </CICardLink>
</CiCards>

> This integration provide useful utilities to hide instable elements from screenshots. Read the dedicated section to learn more about screenshot stability.

Reviewer: ```suggestion
This integration provides useful utilities to hide instable elements from screenshots. Read the dedicated section to learn more about screenshot stability.
```

What is the dedicated section? Link?

## Visual changes detection

Once the [installation](/installation) is done, Argos will receive your application's screenshots when you create a pull request or merge a commit to the repository main's branch.

> Argos compare the screenshots with their baseline counterpart. If a visual change is detected, a red check status is added to the GitHub pull request.

Reviewer: ```suggestion
Argos compares the screenshots with their baseline counterpart. If a visual change is detected, a red check status is added to the GitHub pull request, if everything's right, a green check.
```

<img
  src={githubCheckStatus}
  alt="GitHub check status"
  style={{ borderRadius: 8 }}
/>

> Click on `Details` link to review the differences.

Reviewer: ```suggestion
Click on **"Details"** link to review the differences.
```

## Review the visual changes

> The visual changes review happens on Argos.

Reviewer: ```suggestion
The visual changes review takes place on Argos application.
```

<img src={siriusBuild} alt="Example build" style={{ borderRadius: 8 }} />

> Navigate through the screenshot diffs with the thumbnail list, the arrow buttons or with the keyboard shortcuts.

Reviewer: ```suggestion
Navigate through the screenshot diffs by clicking in sidebar, clicking on the navigation buttons, or using your keyboard.
```

> > Tip: Use the `(?)` shortcut to show the keyboard shortcuts list.

Reviewer: ```suggestion
> Tip: Press `?` to show all available keyboard shortcuts.
```

> The side-by-side UI is designed to easily spot visual changes and verify that screenshots match their expected results.

Reviewer: ```suggestion
The side-by-side UI is designed to easily spots visual changes and verifies that screenshots match their expected results.
```

#### Reject a build

> If you spot a visual bug or if a screenshot doesn't satisfy your expectations, click on `Review changes` button and select `Reject changes`. It will update the build status to `Changes rejected` and notify Github that the test is a failure.

Reviewer: ```suggestion
If you spot a visual bug or if a screenshot doesn't satisfy your expectations, click on "Review changes" button and select "Reject changes". It will update the build status to "Changes rejected" and notify GitHub that changes have been rejected.
```

Backquote are to highlight code, this is not code, use quotes.

> Now that the visual regression is avoided, you can fix the code and commit a new version.

Reviewer: ```suggestion
Now that the changes are rejected, you can fix the code and commit a new version to fix it.
```

#### Approve a build

> When screenshots matches the expected results, click on `Review changes` button then select `Approve changes`. It will update the build status to `Changes approved` and notify Github that the test is passing.

Reviewer: ```suggestion
When screenshots match the expected results, click on `Review changes` button then select `Approve changes`. It will update the build status to `Changes approved` and notify GitHub that the changes have been accepted in Argos.
```

> Now that the visual review is validate, you can merge your pull request with confidence.

Reviewer: ```suggestion
Now that the changes are accepted, you can merge your pull request with confidence.
```

## FAQ

<details>

>   <summary>How to update the screenshot baseline compare ?</summary>

Reviewer: ```suggestion
  <summary>How to update the screenshot baseline compare?</summary>
```

>   When a commit is pushed to the repository's reference branch, Argos update the

Reviewer: ```suggestion
  When a commit is pushed to the repository's reference branch, Argos updates the
```

  comparison baseline with the related screenshots.

> </details>

Reviewer: a `<p>` is missing?

<details>

>   <summary>What is a reference branch ?</summary>

Reviewer: ```suggestion
  <summary>What is a reference branch?</summary>
```

  <p>

>     The reference branch is the branch use as baseline for the screenshot
>     comparisons. Argos detect the Github default branch and use it as reference
>     branch.

Reviewer: ```suggestion
    The reference branch is the branch used as baseline for the screenshot
    comparisons. Argos automatically use GitHub default branch as reference branch. You can choose another one in Argos settings.
```

  </p>
</details>

<details>

>   <summary>How to update the reference branch ?</summary>

Reviewer: ```suggestion
  <summary>How to update the reference branch?</summary>
```

  You can update the reference branch in Argos repository settings.
</details>

<details>

>   <summary>What is a orphan build ?</summary>

Reviewer: ```suggestion
  <summary>What is a orphan build?</summary>
```

>   An Orphan is the first build perform on a reference branch. It wont show any comparison
>   but, the first screenshots will be considered as the new baseline for the next
>   build.

Reviewer: ```suggestion
  An orphan build is the first one performed on a reference branch. It won't show any comparison. The first screenshots uploaded are considered as the baseline for the next
  build.
```

</details>

<details>

>   <summary>What is a reference build ?</summary>

Reviewer: ```suggestion
  <summary>What is a reference build?</summary>
```

  <p>

>     A reference build is created when a commit is merged on the reference
>     branch. It will update the baseline with the related screenshots.

Reviewer: ```suggestion
    A reference build is created when a commit is merged on the reference
    branch. It serves as baseline for the next builds performed on other branches.
```
